### ISSUE TITLE
That's a dick move. (yet I do own license)

### ISSUE THREAD

**Issue by alfuken** [C6H2Cl2/YukariLib#19]
https://github.com/toliner/YukariLib/blob/6057feaf8ee8913b2445db4042f92639336e5886/src/main/java/c6h2cl2/YukariLib/YukariLibCore.kt#L87

And since you call anyone an idiot without even knowing them, it gives that one person a complete indulgence to call you an asshole. You're being a rude, ignorant asshole by doing that.

**Comment by GhostShot3** [comment]
@alfuken Yes exataly, I tried to play with my friend using this mod and this happened too, this a lot of disrespect and lack of consideration for users, @toliner hope you remove it.

**Comment by BadHunyun** [comment]
> You're being a rude, ignorant asshole by doing that.

Not "hope",is "must".
=)

**Comment by burningtnt** [comment]
IT'S IMPOLITE TO ALL THE OFFLINE PLAYER.
You should remove this kind of code in your mod.
